jvm/experiment.yaml

```diff
@@ -62,7 +62,7 @@ spec:
     targetRef:
       apiVersion: batch/v1
       kind: Job
-      name: jvm-reactors
+      name: ""
```

Comment:
Instead of explicitly using the empty string, just omit the `name` field altogether.